codelists/actions.py

```diff
@@ -67,6 +67,31 @@ def create_version(*, codelist, csv_data):
     raise ValueError("E_TOO_MANY_VERSIONS")
 
 
+@transaction.atomic
+def update_codelist(
+    *, codelist, project, name, coding_system_id, description, methodology,
+):
+    """Update a Codelist."""
+
+    codelist.project = project
+    codelist.name = name
+    codelist.coding_system_id = coding_system_id
+    codelist.description = description
+    codelist.methodology = methodology
+
+    codelist.save(
+        update_fields=[
```

Comment:
TIL.

Reply:
Not sure it's necessary though, and means extra work if we add another field.

Reply:
That's true, this is more a habit of mine after working in some databases where tables had large fields.  I'll remove this as I don't think it adds anything performance or readability wise!

Reply:
Fixed in 7abe19b